web/src/main/webapp/resources/css/design.css

```diff
@@ -6102,9 +6099,9 @@ table.pointDetails td.pointDetailsValue {
 }
 
 input.rf-au-inp {
-    width: 206px !important;
+    width: 206px;
     border: 1px solid #CCCCCC !important;
-    padding: 4px !important;
+    padding: 4px;
```

Comment:
I don't think so. Well, it depends how you use it. If you don't provide any other class like input-xlarge, than there will be a default width. I'm not sure if there is such case anywhere in the application, therefore I prefer not removing it.